### PR TITLE
Add rubocop style enforcement to the CI build

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: tasks/hydra-collections-dev.rake [hunk 1]
```diff
@@ -11,6 +18,8 @@ end
 
 desc 'Start solr, fcrepo, and run specs'
 task ci: ['engine_cart:generate'] do
```
Comment (mjgiarlo) [inline tasks/hydra-collections-dev.rake]: I wonder if we should add the rubocop task here instead to fail quickly (before EC does its thing) if there are style issues.

**Comment by mjgiarlo** [comment]
@cbeer :cop: :red_circle: 

```
Offenses:

app/models/concerns/hydra/collection.rb:13:40: C: Style/ZeroLengthPredicate: Use empty? instead of size < 1.

      return if new_member_ids.nil? || new_member_ids.size < 1
                                       ^^^^^^^^^^^^^^^^^^^^^^^

37 files inspected, 1 offense detected
RuboCop failed!
```

File: tasks/hydra-collections-dev.rake (hunk 1 above)
Comment (cbeer) [inline tasks/hydra-collections-dev.rake]: :shrug:

**Comment by cbeer** [comment]
Must be a difference between rubocop versions. I'll add more specific pinning, I suppose.

**Pull request merged**